### DISCUSSION TITLE
Gremlin 3.6.4

### DISCUSSION THREAD

**Discussion by anheri** [Gremlinq/ExRam.Gremlinq#1035] [General]
Tried to upgrade to Gremlin 3.6.4, but then Gremlinq failed... 
Will 3.6.4 be supported in the near future?

**Comment by danielcweber** [comment]
What's the reason for explicitly overriding the transitive dependency on Gremlin.NET?

**Reply by anheri** [reply]
I use gremlin queries where gremlinq is lacking functionality... so I pull in the gremlin.net package aswell

**Comment by danielcweber** [comment]
You realize it's already there through ExRam.Gremlinq (although in a lower version)?

**Reply by anheri** [reply]
yes, I should probably remove it...

**Comment by danielcweber** [comment]
I guess this is resolved.
